package-lock.json

```
@@ -25,7 +25,7 @@
                 "@fortawesome/fontawesome-svg-core": "1.2.36",
                 "@fortawesome/free-regular-svg-icons": "5.15.4",
                 "@fortawesome/free-solid-svg-icons": "5.15.4",
-                "@ls1intum/apollon": "2.8.3",
+                "@ls1intum/apollon": "^2.8.4",
```

Use fixed versions to avoid sudden failures by automatic updates
```suggestion
                "@ls1intum/apollon": "2.8.4",
```